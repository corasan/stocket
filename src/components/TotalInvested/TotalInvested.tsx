import { View } from 'react-native'

export const TotalInvested = () => <View />

export default TotalInvested
