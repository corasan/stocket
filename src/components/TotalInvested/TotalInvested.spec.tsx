import React from 'react'
import { render } from '@testing-library/react-native'
import TotalInvested from './TotalInvested'

describe('TotalInvested', () => {
  it('matches the snapshot', () => {
    const { toJSON } = render(<TotalInvested />)

    expect(toJSON()).toMatchSnapshot()
  })
})
