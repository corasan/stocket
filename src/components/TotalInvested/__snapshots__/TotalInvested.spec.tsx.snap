// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalInvested matches the snapshot 1`] = `<View />`;
